internal/skeleton/skeleton_test.go

```diff
@@ -1,108 +0,0 @@
-package skeleton
-
-import (
-	"os"
-	"path/filepath"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
-)
-
-func TestIsInsideSkeletonDir(t *testing.T) {
```

Comment:
Some of the skeleton tests were removed and need to be added again. Maybe a good chance to rewrite them.